### PR TITLE
Issue: Windows commands not accessible inside distribution terminal

### PR DESCRIPTION
**Pull request by dimithras**
Follow up on #880.
Referencing @craigloewen-msft [comment](https://github.com/MicrosoftDocs/WSL/pull/880#issuecomment-700145356) considering all requested changes. Double checked for typos and wording, should be safe to merge.

New PR made as I accidentally deleted the fork branch for #880.

For further details please see #880 and original issue #878.

### CODE REVIEW

**Comment by dimithras** [comment]
Ready for review

File: WSL/troubleshooting.md [hunk 1]
```diff
@@ -324,4 +324,15 @@ enabled = true
 options = metadata,uid=1000,gid=1000,umask=0022
 ```
 
-Please note that adding this command will include metadata and modify the file permissions on the Windows files seen from WSL. Please see the [File System Permissions](./file-permissions.md) for more information.
\ No newline at end of file
+Please note that adding this command will include metadata and modify the file permissions on the Windows files seen from WSL. Please see the [File System Permissions](./file-permissions.md) for more information.
+
+### Running Windows commands fails inside a distribution
+
+Some distributions [available in Microsoft Store](install-win10.md#step-6---install-your-linux-distribution-of-choice) are yet not fully compatible to run Windows commands in [Terminal](https://en.wikipedia.org/wiki/Linux_console) out of the box. If you get an error `-bash: powershell.exe: command not found` running `powershell.exe /c start .` or any other Windows command, you can resolve it following these steps:
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md:331]:
```suggestion
If you get an error when running Windows commands such as: `-bash: powershell.exe: command not found` running `powershell.exe /c start .` or any other Windows command, you can resolve it following these steps:
```

File: WSL/troubleshooting.md [hunk 2]
```diff
@@ -324,4 +324,15 @@ enabled = true
...
+
+1. In wsl distribution Terminal run `echo $PATH`.
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md:333]:
```suggestion
1. In your WSL distribution run `echo $PATH`.  
```

File: WSL/troubleshooting.md [hunk 3]
```diff
@@ -324,4 +324,15 @@ enabled = true
...
-Please note that adding this command will include metadata and modify the file permissions on the Windows files seen from WSL. Please see the [File System Permissions](./file-permissions.md) for more information.
\ No newline at end of file
+Please note that adding this command will include metadata and modify the file permissions on the Windows files seen from WSL. Please see the [File System Permissions](./file-permissions.md) for more information.
+
+### Running Windows commands fails inside a distribution
+
+Some distributions [available in Microsoft Store](install-win10.md#step-6---install-your-linux-distribution-of-choice) are yet not fully compatible to run Windows commands in [Terminal](https://en.wikipedia.org/wiki/Linux_console) out of the box. If you get an error `-bash: powershell.exe: command not found` running `powershell.exe /c start .` or any other Windows command, you can resolve it following these steps:
+
+1. In wsl distribution Terminal run `echo $PATH`.  
+   If it does not feature `/mnt/c/Windows/system32` something is redefining the standard PATH variable.
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md:334]:
```suggestion
   If it does not include: `/mnt/c/Windows/system32` something is redefining the standard PATH variable.
```

File: WSL/troubleshooting.md [hunk 4]
```diff
@@ -324,4 +324,15 @@ enabled = true
...
+2. Check profile settings with `cat /etc/profile`.  
+   If it contains assignment of PATH variable, run this command `sudo sed -i 's/PATH/NULL/g' /etc/profile` or edit manually the file with one of Linux Terminal text editors, ie `sudo nano /etc/profile`, commenting out PATH block with **#** character.
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md:336]:
```suggestion
   If it contains assignment of the PATH variable, edit the file to comment out PATH assignment block with a **#** character.
```

**Comment by craigloewen-msft** [comment]
Added in some small edits and merged this into master! Thank you for the contribution! :)

**Pull request merged**